docs/source/modin/dataframe.rst

```diff
@@ -77,7 +77,6 @@ DataFrame
     DataFrame.rfloordiv
     DataFrame.rmod
     DataFrame.rpow
-    DataFrame.round
```

Comment:
we implemented round already cc @sfc-gh-helmeleegy

Reply:
It's still in the doc, this was just a duplicate entry.